src/templates/json-editor.html

```diff
@@ -11,9 +11,9 @@ <h6 id='jsonEditorTitle'></h6>
     {% if mode == 'edit' %}
       <div class='mt-4 text-center'>
         <div class='btn-group'>
-          <button type='button' data-action='json-save' class='btn btn-primary'>{{ 'Save'|trans }}</button>
+          <button type='button' id='json-save' data-action='json-save' class='btn btn-primary'>{{ 'Save'|trans }}</button>
           <div class='btn-group dropright' role='group'>
-            <button type='button' class='btn btn-primary dropdown-toggle dropdown-toggle-split' data-toggle='dropdown' aria-haspopup='true' aria-expanded='false'>
+            <button type='button' id='json-save-dropdown' class='btn btn-primary dropdown-toggle dropdown-toggle-split' data-toggle='dropdown' aria-haspopup='true' aria-expanded='false'>
```

Comment:
and here

Reply:
This however is used in the save() method in JsonEditorHelper.
And there are some dropdowns in the code base so it might be better to use an ID?

Reply:
where is it used in `save()` ? I see `#json-save-dropdown`.

Reply:
Maybe I don't get your point but isn't this the way to use an [ID as a selector in jQuery](https://api.jquery.com/id-selector/)?
I tried to avoid/remove jQuery stuff but with the `.is(':hidden')` and `.dropdown('toggle')` I don't know how for now.

Reply:
Yep, not sure what I was saying, no problem here.